app/jobs/score_updater_job.rb

```diff
@@ -0,0 +1,22 @@
+class ScoreUpdaterJob
+  include SuckerPunch::Job
+
+  def perform(user_id, score=1)
+    add_score(user_id, score, DailyScore)
+    add_score(user_id, score, WeeklyScore)
+    add_score(user_id, score, MonthlyScore)
+    add_score(user_id, score, OverallScore)
+  end
+
+  def add_score(user_id, score, score_klass, expires_at=nil)
```

Comment:
Cool @bugant, I'm open to this new architecture proposal, please show me the way...